src/tpm2_getmanufec.c

```diff
@@ -593,21 +542,25 @@ int main(int argc, char *argv[])
             strncpy(EKserverAddr, optarg, strlen(optarg));
             printf("TPM Manufacturer EK provisioning address -- %s\n", EKserverAddr);
             break;
-        default:
-            showHelp(argv[0]);
-            return -10;
         }
     }
 
-    prepareTest(hostName, port, debugLevel);
-
-    if (!OfflineProv)
-        returnVal                         = createEKHandle();
-
-    ProvisioningreturnVal = TPMinitialProvisioning();
-
-    finishTest();
-
+    int returnVal = 1;
+    int ProvisioningreturnVal = 0;
+    if(argc < 2)
+    {
```

Comment:
space between if and (, also brace on same line as if, this goes throughout your changes